src/engraving/libmscore/textlinebase.cpp

```diff
@@ -602,6 +603,21 @@ TextLineBase::TextLineBase(const ElementType& type, EngravingItem* parent, Eleme
 {
     setBeginHookHeight(Spatium(1.9));
     setEndHookHeight(Spatium(1.9));
+    setGapBetweenTextAndLine(Spatium(0.5));
+}
+
+//---------------------------------------------------------
+//   write
+//---------------------------------------------------------
+
+void TextLineBase::write(XmlWriter& xml) const
+{
+    if (!xml.context()->canWrite(this)) {
+        return;
+    }
+    xml.startElement(this);
+    writeProperties(xml);
+    xml.endElement();
```

Comment:
The `write` method for individual elements has been removed recently; now, writing happens in the `TWrite` class. I don't expect you will need to add anything there. However, it would be good to rebase on top of the latest master branch (`git pull --rebase upstream master` if `upstream` is the name of the remote pointing to `https://github.com/musescore/MuseScore.git`), since there have been quite significant changes during the last few days.

Reply:
Just learned that one the hard way, rebasing and trying again. Thanks!